nbgrader/utils.py

```diff
@@ -215,6 +216,39 @@ def is_ignored(filename, ignore_globs=None):
             return True
     return False
 
+def ignore_patterns(exclude=None, include=None, max_file_size=None, log=None):
```

Comment:
Nice!

Would you mind adding an args and returns section to the docstring?

Reply:
Done!